lib/Service/src/SettingsHandler.cpp

```diff
@@ -150,6 +156,21 @@ bool SettingsHandler::loadConfigurationFile(const String& filename)
                 m_platoonVehicleId = jsonPlatoonVehicleId.as<uint8_t>();
             }
 
+            if (false == jsonInitialXPosition.isNull())
+            {
+                m_initialXPosition = jsonInitialXPosition.as<int32_t>();
+            }
+
+            if (false == jsonInitialYPosition.isNull())
+            {
+                m_initialYPosition = jsonInitialYPosition.as<int32_t>();
+            }
+
+            if (false == jsonInitialHeading.isNull())
+            {
+                m_initialHeading = jsonInitialHeading.as<int32_t>();

```

Comment:
Wasn't it a 16 bit value?

Reply:
No, heading is a 32 bit value. Motor speeds are 16 bit